src/firestore/delete.ts

```diff
@@ -0,0 +1,486 @@
+"use strict";
```

Comment:
you can drop use strict in ts files.... wonder if there's a lint rule for that...